ui/views/components/head.ejs

```diff
@@ -0,0 +1,72 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<meta name="theme-color" content="#235DF4" />
+<meta name="description" content="" />
+
+<title>DanaSwap App</title>
+
+<link rel="icon" href="./assets/images/favicon.ico" />
+
+<link rel="stylesheet" id="theme-css" href="./assets/styles/night.css" />
+
+<link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
```

Comment:
We should not be trusting a CDN ... especially not since we are handling financial data. Security conscious folks can and will be blocking third-party fonts, and this means these will break. Also, icon fonts are bad for a gajillion reasons. We should be using some for of SVG spriting (usually `<symbols>`) for icons IMO.

Reply:
good point, will download the CSS and make it self-hosted

Reply:
And use a non-font icon system

Reply:
I am wondering why not?

Reply:
A bit of an older article, but CSS Tricks generally stuff I can trust https://css-tricks.com/icon-fonts-vs-svg/

Also privacy/security circles tend to publish articles like this https://collinmbarrett.com/block-web-fonts/ that suggest at a minimum blocking all third-party fonts, but some suggest all fonts together if you want a faster experience (this is why I always suggest having the correct "fallback" font). These users are getting broken boxes in their browsers because they don't want their data being leaked to these services like Google Fonts, but also Font Awesome. The way we blindly included all of Font Awesome too ... well, we're shipping 1000s of icons when we're only using like <20 which is a lot of unused data being sent. It's 2022--CDNs shouldn't be trusted, and one of the old reasons to use them was shared caches of content but all the latest browsers are doing first-party isolation so there's no performance gain and only leaked data for users.

You weren't there for Costa Rica, but I gave an hour-long talk about the advantages of designing a good experience for these types of folks because everyone benefits. And when you think about it: icons are a graphic, an image; they are not a font. Even philosophically it feels like the right tool for the job too.

The history of the icon font goes back to needing to support a vectorized image that worked across platforms--particularly IE6 and 7 with `.eot` font files. These old browsers, the SVG support was not good. With `currentColor` landing in I think IE9 and SVG support getting good about then too, folks circled back to SVGs and `<symbol>` s, like the article I sent you, because it got them all the features they needed while picking something that's easier to build (building fonts is resource-intensive and you can't just add a new symbol like an SVG but instead have to recompile the font).